src/types/cards.types.ts

```diff
@@ -0,0 +1,8 @@
+export type Card = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<"svg">>;
+  list: (string | JSX.Element)[][];
```

Comment:
you can use `React.ReactNode` instead of JSX.Element

it also includes string type